nbbuild/templates/projectized.xml

```diff
@@ -810,7 +810,8 @@ If you are sure you want to build with JDK 9+ anyway, use: -Dpermit.jdk9.builds=
         <delete file="${report.file}" failonerror="false"/>
     </target>
 
-    <target name="patch-for-maven" depends="init" description="Patch local Maven repo's -DmavenVersion of this module with one built from sources.">
+    <target name="patch-for-maven" depends="maven-install"/> <!-- deprecated -->
```

Comment:
@jglick, I was always wondering what `patch-for-maven` target stands for. After many years I finally took a look and found out that it is the task I was always looking for. Probably a sign the target's name isn't best. I propose to name it `maven-install`.